<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_order_window</class>
 <widget class="QDialog" name="add_order_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>249</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>409</width>
    <height>249</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>409</width>
    <height>249</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Order Tracker - Add Order</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Quanitity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Owner</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="item_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="quantity_spin_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.100000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="owner_combo_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PSS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>All Residents</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="status_combo_box">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Outstanding</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Complete</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="description_combo_box">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>261</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nutritional Supplement</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Liquids (mls)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dressings</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Injections</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Capsules</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nebules</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sachets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Creams</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Caplets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tablets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Feeds</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Other</string>
    </property>
   </item>
  </widget>
 </widget>
 <tabstops>
  <tabstop>owner_combo_box</tabstop>
  <tabstop>item_edit</tabstop>
  <tabstop>description_combo_box</tabstop>
  <tabstop>quantity_spin_box</tabstop>
  <tabstop>status_combo_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>add_order_window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>add_order_window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
